src/components/header.css

```diff
@@ -0,0 +1,63 @@
+.header {
+  width: 100%;
+  height: 100px;
+  position: fixed;
+}
+
+.headerContainer {
+  display: flex;
+  justify-content: space-between;
+  margin: 0 10rem;
+  height: 100px;
+  align-items: center;
+}
+
+.navList {
+  margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.linkLi {
+  list-style: none;
+  margin: 0;
+  margin-left: 1rem;
+  letter-spacing: 1.25px;
+  text-transform: uppercase;
+  font-size: 16px;
+}
+
+.linkLi:hover {
+  cursor: pointer;
+}
+
+.navLink {
+  font-family: "Visuelt Bold", sans-serif;
```

Comment:
Can we add font styles to our global css sheet? It would be good to have most of that setup there so we can knock out as many use cases as possible, and then setup a variable if we need to modify specific elements.